assets/js/util.js

```diff
@@ -119,6 +119,27 @@ $(document).ready(function () {
     $('.modal').on('hide.bs.modal', function () {
         $('a[data-target=#' + $(this).attr('id') + ']').parents('li').removeClass('active');
     });
+
+    $(window).scroll(function () {
+        if($(window).scrollTop() > BACK_TO_TOP_ACTIVATION_HEIGHT) {
+            $('#backToTop')
```

Comment:
Ah, this is a better approach. However, using a conditional expression like this is an anti-pattern. Use `toggleClass` for the cleanest code.